.devcontainer/devcontainer.json

```diff
@@ -1,7 +1,22 @@
 {
   "image": "mcr.microsoft.com/devcontainers/python:3.11",
-  "features": { "ghcr.io/devcontainers/features/desktop-lite:1": {}
+  "features": { 
+	"ghcr.io/devcontainers/features/desktop-lite:1": {},
+	"ghcr.io/devcontainers/features/github-cli:1": {}
```

Comment:
Excellent addition, just one question I am concerned about. This PullRequest will use multiple DockerImage, will there be any conflicts? I would catch any information on this.

Reply:
Sorry. It's my misunderstanding. It features, not images so there is no conflict.

Reply:
Features are a recent addition to devcontainers.  Something that previously required more complex docker container handling, and one reason I was attracted to Gitpod previously.  The ones here are supported by GitHub directly, which is important for maintainability.